coops-ui/src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -35,15 +35,28 @@
       <v-container id="standalone-office-address-container" class="view-container">
         <article id="standalone-office-address-article">
           <header>
-            <h1 id="filing-header">Address Change</h1>
+            <h1 id="filing-header">Change of Office Addresses</h1>
+            <p>Please change your Registered Office Address
+              <span v-if="entityFilter(EntityTypes.BCorp)">and Record Address</span>
+            </p>
+            <v-alert
+              v-if="entityFilter(EntityTypes.BCorp)"
+              type="info"
+              :value="true"
+              icon="mdi-information"
+              outlined
+              class="white-background">
+              Any address update will be effective tomorrow.
+            </v-alert>
           </header>
 
-          <!-- Registered Office Addresses -->
+          <!-- Office Addresses -->
           <section>
-            <RegisteredOfficeAddress
+            <OfficeAddresses
               :changeButtonDisabled="false"
-              :legalEntityNumber="entityIncNo"
               :addresses.sync="addresses"
+              :registeredAddress.sync="registeredAddress"
+              :recordsAddress.sync="recordsAddress"
```

Comment:
This is confusing. If I understand correctly, for Coop I use 'addresses' but for BCorp I use 'registeredAddress' and 'recordsAddress'??? Um, can we either use a single set of properties, or at least make this much clearer (eg, by renaming the props)?

Reply:
Both entities use Addresses as it is used to retrieve filings, from a draft for instance. 
The registered and records props are what is retrieved from the state providing our baseline addresses, in case a user wants to reset it default.
I could definitely try and make this a little clearer, would some comments help perhaps?

Reply:
Both entities use Addresses as it is used to retrieve filings, from a draft for instance. 
The registered and records props are what is retrieved from the state providing our baseline addresses, in case a user wants to reset it default.
I could definitely try and make this a little clearer, would some comments help perhaps?

Reply:
I'm sitll confused, so yes please, more code comments!

Reply:
Expanded comments inside of the `OfficeAddresses` component where the props are declared :)

Reply:
Thanks.